### DISCUSSION TITLE
Add support for 3D position in graph drawing

### DISCUSSION THREAD

**Discussion by lobpcg** [networkx/networkx#7145] [Ideas]
The present functionality for graph drawing is great with lots of features. Unfortunately, it doesn't support drawing in 3D, giving an error if attempted. Thus, e.g., all recently added examples in the Gallery of 3D drawing need to call matplotlib directly rather than simply using the NetworkX native code. The position functions do apparently support any number of dimensions, the drawing function itself needs to be expanded to 3D.

**Comment by lobpcg** [comment]
Appears to be already raised in https://github.com/networkx/networkx/discussions/6803

**Reply by rossbar** [reply]
Indeed - let's keep the discussion focused there. Please feel free to add to #6803!
